src/calculateRentalCost.js

```diff
@@ -28,7 +28,23 @@
  */
 
 function calculateRentalCost(days) {
-  // write code here
+  const PRICE_PER_DAY = 40;
+  const LONG_TERM = 7;
+  const LONG_TERM_DISCOUNT = 50;
+  const SHORT_TERM = 3;
```

Comment:
The constant SHORT_TERM is not descriptive enough, it could be renamed to something like MIN_DAYS_FOR_SHORT_TERM_DISCOUNT.

Reply:
> The constant SHORT_TERM is not descriptive enough, it could be renamed to something like MIN_DAYS_FOR_SHORT_TERM_DISCOUNT.

Thanks, I've already fixed it